From c391b6a3249e74d64cc99331155f4fa6c04202ea Mon Sep 17 00:00:00 2001
From: Dinolek <git@dinolek.me>
Date: Wed, 9 Jul 2025 17:05:39 +0200
Subject: [PATCH 044/115] dt-bindings: pwm: mediatek,pwm-disp: Add support for
 MT6785

Add mediatek,mt6785-disp-pwm compatible with mediatek,mt8183-disp-pwm
fallback.

Signed-off-by: Dinolek <git@dinolek.me>
---
 Documentation/devicetree/bindings/pwm/mediatek,pwm-disp.yaml | 1 +
 1 file changed, 1 insertion(+)
 mode change 100644 => 100755 Documentation/devicetree/bindings/pwm/mediatek,pwm-disp.yaml

diff --git a/Documentation/devicetree/bindings/pwm/mediatek,pwm-disp.yaml b/Documentation/devicetree/bindings/pwm/mediatek,pwm-disp.yaml
old mode 100644
new mode 100755
index 68ef30414..44a7717d4
--- a/Documentation/devicetree/bindings/pwm/mediatek,pwm-disp.yaml
+++ b/Documentation/devicetree/bindings/pwm/mediatek,pwm-disp.yaml
@@ -27,6 +27,7 @@ properties:
           - const: mediatek,mt8173-disp-pwm
       - items:
           - enum:
+              - mediatek,mt6785-disp-pwm
               - mediatek,mt6893-disp-pwm
               - mediatek,mt8186-disp-pwm
               - mediatek,mt8188-disp-pwm
-- 
2.56.0

